components/eamxx/src/share/atm_process/atmosphere_process.cpp

```diff
@@ -498,6 +545,35 @@ add_precondition_check (const prop_check_ptr& pc, const CheckFailHandling cfh)
 void AtmosphereProcess::
 add_postcondition_check (const prop_check_ptr& pc, const CheckFailHandling cfh)
 {
+  auto cfh2str = [] (const CheckFailHandling cfh) -> std::string {
+    std::string s = "";
+    switch (cfh) {
+      case CheckFailHandling::Fatal:
+        s = "Fatal";
+        break;
+      case CheckFailHandling::Warning:
+        s = "Warning";
+        break;
+      default:
+        EKAT_ERROR_MSG ("Unexpected/unsupported CheckFailHandling value.\n");
+    }
+
+    return "";
```

Comment:
why are we returning anything for this `void` function?

Reply:
Yeah, that's not right. I'll fix it.

Reply:
Oh, no, wait, that's right. The return is for the lambda function, not the class method.